plugins/GSdx/GSTextureCache.cpp

```diff
@@ -403,6 +403,33 @@ GSTextureCache::Source* GSTextureCache::LookupSource(const GIFRegTEX0& TEX0, con
 	return src;
 }
 
+void GSTextureCache::ScaleTexture(GSTexture* texture)
+{
+	if (m_renderer->CanUpscale())
```

Comment:
Potentially it would be nice to replace this gigantic scoped if
```
if (m_renderer->CanUpscale())
{


*stuffs*

}
```

with 

```
if (!m_renderer->CanUpscale()) return;
```

Any opinions?